envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -538,11 +563,13 @@ def test_depth_4_a(self, island_client):
     # Not grouped because it's slow
     def test_zerologon_exploiter(self, island_client):
         test_name = "Zerologon_exploiter"
+
```

Comment:
No need for these spaces

Reply:
It makes the test easier to understand. I'd like to leave them in.